src/monopoly/monopoly_player.js

```diff
@@ -75,13 +71,15 @@ MonopolyPlayer.prototype.owesRent = function(property) {
 };
 
 MonopolyPlayer.prototype.owns = function(property) {
-  return property.owner === this;
+  return property.owner && property.owner.id === this.id;
```

Comment:
Do you want to merge it or wait for me to fix those few things? I'm going to do some grading for the next hour or so first